src/sentry/services/hybrid_cloud/organization/impl.py

```diff
@@ -138,6 +138,23 @@ def get_org_by_slug(
         except Organization.DoesNotExist:
             return None
 
+    def get_org_by_id(
```

Comment:
@iamrajjoshi We have `organization_service.get_organization_by_id()` that already does this: https://github.com/getsentry/sentry/blob/b3f8ab3aea7eaabfd408314997f8c21c5707491f/src/sentry/services/hybrid_cloud/organization/impl.py#L95-L122

Reply:
It doesn't return the same object and doesn't take visibility status in account.